test/dummy/config/initializers/mountain_view.rb

```diff
@@ -1,3 +1,6 @@
 MountainView.configure do |config|
   config.included_stylesheets = ["global"]
 end
+
+# include component classes
+Dir.glob('app/components/**/*.rb') { |f| require_relative "../../#{f}" }
```

Comment:
Totally agree, to be honest I'd put that in temporarily and totally forgotten about it. Thanks for the pointer.